server/src/modules/mentorship-response/mentorship-response.model.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Model,
+  Column,
+  Table,
+  DataType,
+  ForeignKey,
+  PrimaryKey,
+  AutoIncrement,
+} from 'sequelize-typescript';
+import { MentorshipRequest } from '../mentorship-request/mentorship-request.model';
+
+@Table({ tableName: 'mentorship_response' })
+export class MentorshipResponse extends Model<MentorshipResponse> {
+  private static ACCEPTED = 'Accepted';
+  private static DECLINED = 'Declined';
+
+  private static MENTORSHIP_RESPONSE_STATUS = DataType.ENUM(
+    MentorshipResponse.ACCEPTED,
+    MentorshipResponse.DECLINED,
+  );
+
+  @PrimaryKey
+  @AutoIncrement
+  @Column(DataType.INTEGER)
+  id: number;
+
+  @ForeignKey(() => MentorshipRequest)
```

Comment:
(nit: ideally might be nice to make this unique, but I can't find an easy way to do it in Sequelize so probably fine to skip)

Reply:
This can be done with indexes.